ros/api/v1/hosts.py

```diff
@@ -21,6 +21,25 @@
 SYSTEM_STATES_EXCEPT_EMPTY = [
     "Oversized", "Undersized", "Idling", "Under pressure", "Storage rightsizing", "Optimized", "Waiting for data"
 ]
+OS_VERSIONS = {
+    '7.0': {"name": "RHEL", "major": 7, "minor": 0},
+    '7.1': {"name": "RHEL", "major": 7, "minor": 1},
+    '7.2': {"name": "RHEL", "major": 7, "minor": 2},
+    '7.3': {"name": "RHEL", "major": 7, "minor": 3},
+    '7.4': {"name": "RHEL", "major": 7, "minor": 4},
+    '7.5': {"name": "RHEL", "major": 7, "minor": 5},
+    '7.6': {"name": "RHEL", "major": 7, "minor": 6},
+    '7.7': {"name": "RHEL", "major": 7, "minor": 7},
+    '7.8': {"name": "RHEL", "major": 7, "minor": 8},
+    '7.9': {"name": "RHEL", "major": 7, "minor": 9},
+    '8.0': {"name": "RHEL", "major": 8, "minor": 0},
+    '8.1': {"name": "RHEL", "major": 8, "minor": 1},
+    '8.2': {"name": "RHEL", "major": 8, "minor": 2},
+    '8.3': {"name": "RHEL", "major": 8, "minor": 3},
+    '8.4': {"name": "RHEL", "major": 8, "minor": 4},
+    '8.5': {"name": "RHEL", "major": 8, "minor": 5}
+}
```

Comment:
@apuntamb Shall we move this into the `constants.py` file?

Reply:
@saltgen For now, let's keep this constant along with the others in the same file. We can consider refactoring in a separate PR.